proto/self_serve/app/v1/app.proto

```diff
@@ -2,6 +2,16 @@ syntax = "proto3";
 
 package self_serve.app.v1;
 
+// Wrapper message for all messages that can be sent to and from the app. The reason we do this is to ensure static
+// typing in the clients while the server can still handle all messages through the Any type.
+message W {
```

Comment:
Should we make it a bit more verbose than `W`, `AppMessageWrapper` or `Wrapper` or `Envelope`?

Reply:
Actually I was trying to make it as less verbose as possible to follow MongoDB's types: https://github.com/worldcoin/signup-service/blob/main/internal/user/session/store.go#L132

😆 Let me know if you don't like it! More than happy to make it verbose!

Reply:
fortunately non of the naming affects the wire format, so we can change it whenever